cmd/lotus-miner/run.go

```diff
@@ -55,10 +57,12 @@ var runCmd = &cli.Command{
 			}
 		}
 
+		network := strings.Split(buildconstants.BuildTypeString(), "+")[1]
 		ctx, _ := tag.New(lcli.DaemonContext(cctx),
 			tag.Insert(metrics.Version, build.MinerBuildVersion),
 			tag.Insert(metrics.Commit, build.CurrentCommit),
 			tag.Insert(metrics.NodeType, "miner"),
+			tag.Insert(metrics.Network, network),
```

Comment:
```suggestion
		ctx, _ := tag.New(lcli.DaemonContext(cctx),
			tag.Insert(metrics.Version, build.MinerBuildVersion),
			tag.Insert(metrics.Commit, build.CurrentCommit),
			tag.Insert(metrics.NodeType, "miner"),
			tag.Insert(metrics.Network, buildconstants.NetworkBundle),
```

not really ideal, but this is the value we want and it's a bit safer than manipulating the string that's intended for version printing